src/app/components/shell/header/menus/main-menu/main-menu.js

```diff
@@ -17,7 +17,7 @@ const menuStructure = [
         label: 'Technology',
         items: [
             {label: 'OpenStax Kinetic', url: '/kinetic'},
-            {label: 'OpenStax Assignable', url: '/assignable'},
+            // {label: 'OpenStax Assignable', url: '/assignable'},
```

Comment:
saving for later?

Reply:
Yes, it will be re-enabled next release.